# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/printing/content/printUtils.js
# Commit: f95c19e45ab2
# Full Hash: f95c19e45ab2ef6a1ba8ac8d4124df117928b1a4
# Author: Jonathan Watt <jwatt@jwatt.org>
# Date: 2020-07-22 21:55:45
# Regressor Bug: 1652270
# File Overlap Count: 1
# Description:
#   Bug 1652270. Convert nsFrameLoader::Print to return a Promise. r=farre,remote-protocol-reviewers,marionette-reviewers,jgraham,whimboo,mixedpuppy
#   
#   (Instead of requiring callers to pass an nsIWebProgressListener.)
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D83249
# ==============================================================================

diff -r c749c42dc6b9 -r f95c19e45ab2 toolkit/components/printing/content/printUtils.js
--- a/toolkit/components/printing/content/printUtils.js	Wed Jul 22 18:40:32 2020 +0300
+++ b/toolkit/components/printing/content/printUtils.js	Wed Jul 22 15:54:01 2020 +0000
@@ -182,6 +182,8 @@
       printSettings.docURL = this._originalURL;
     }
 
+    // At some point we should handle the Promise that this returns (report
+    // rejection to telemetry?)
     topBrowser.print(windowID, printSettings);
 
     if (printPreviewIsOpen) {